admin/pages/accounts.php

```diff
@@ -457,7 +459,7 @@ class="fa fa-remove"></i> Cancel</span></a>
     <?php
     if (isset($account) && $account->isLoaded()) {
         $account_players = array();
-        $query = $db->query('SELECT `name`,`level`,`vocation`  FROM `players` WHERE `account_id` = ' . $account->getId() . ' ORDER BY `name`')->fetchAll();
+        $query = $db->query('SELECT name,level,vocation  FROM players WHERE account_id = ' . $account->getId() . ' ORDER BY name')->fetchAll();
```

Comment:
why are you removing **`** and spaces ?

Reply:
I removed it just to clean up the code, as I'm used to programming this way, I ended up forgetting to follow the pattern here.